packages/docusaurus-init/templates/classic/src/css/custom.css

```diff
@@ -1,3 +1,10 @@
+/**
```

Comment:
We don't really want to add this rule to the non-FB templates because they are... non-FB. It's ok I'll remove it later on.

Reply:
Good to know, thanks!